README.md

```diff
@@ -295,8 +295,6 @@ Arm welcomes feedback on the design of the API. If you think something could be
 Mbed TLS includes a reference implementation of the PSA Cryptography API.
 However, it does not aim to implement the whole specification; in particular it does not implement all the algorithms.
 
```

Comment:
There's [still one dangling reference](https://github.com/Mbed-TLS/mbedtls/pull/9818#pullrequestreview-2502614277) in `ChangeLog.d/psa-always-on.txt`, to `use-psa-crypto.md` which no longer exists.